notebooks/Makefile

```diff
@@ -16,3 +16,4 @@ test-nb: lint unittest
 
 unittest:
 	pytest tests
+	rm -rf tmp
```

Comment:
The `fs` notebook creates lots of copies, symbolic links, and empty directories to files that don't need to be saved, so I place them in a `tmp` directory and clean them up after unit testing.

Reply:
FYI about `make` and `Makefile`s: In a multiline recipe like this, the first failing command terminates the recipe, so if `pytest tests` fails (exits with non-zero status, like if a test fails), `rm -rf tmp` won't run. If a subsequent test run depends on these files having been removed, that might be a problem -- in which case doing the `rm` first might be necessary. But if the tests just overwrite their previous output, this might not be needed at all, since it's already `.gitignore`d. There are similar temporary files created by `make docs` that we don't even try to delete, because `git` has been told to ignore them and they don't make any trouble. Your call, and you might have reasons I'm not thinking of for doing this `rm`, but it might not be needed.